common/src/main/java/com/amazonaws/services/schemaregistry/common/AWSSchemaRegistryClient.java

```diff
@@ -90,11 +91,26 @@ public AWSSchemaRegistryClient(@NonNull AwsCredentialsProvider credentialsProvid
                 .addExecutionInterceptor(new UserAgentRequestInterceptor())
                 .build();
 
+        UrlConnectionHttpClient.Builder urlConnectionHttpClientBuilder = UrlConnectionHttpClient.builder();
+
+        if (glueSchemaRegistryConfiguration.getProxyUrl() != null) {
+        	log.debug("Creating http client using proxy {}", glueSchemaRegistryConfiguration.getProxyUrl());
+        	try {
+    	    	URI uri = new URI(glueSchemaRegistryConfiguration.getProxyUrl());
+    	        ProxyConfiguration proxy = ProxyConfiguration.builder().endpoint(uri).build();
+    	        urlConnectionHttpClientBuilder.proxyConfiguration(proxy);
+        	} catch (URISyntaxException e) {
+        		String message = String.format("Malformed uri, please pass the valid proxy uri for creating the client",
+                        glueSchemaRegistryConfiguration.getEndPoint());
```

Comment:
This error message seems incorrect. Should this be `.getProxyUrl` instead? Also, the `String.format` parameters must have a placeholder `%s' for printing the Uri.

Reply:
fixed it